### DISCUSSION TITLE
Should `isotopicDepletion` and `fissionProductModel` be one package?

### DISCUSSION THREAD

**Discussion by albeanth** [terrapower/armi#1504] [General]
I have not thought about this critically, but during a PR review found myself asking why they are separate packages. They are both related to depletion and could likely be coalesced into a single depletion package. The isotopicDepletion package has no tests, which is also suspect (though I suppose there are tests in the parent directory, which is kind of odd). I think combining them and maybe doing some potential cleanup would help improve long term maintainability of both packages.

**Comment by john-science** [comment]
I'm all for better code organization. This is one of those that will _probably_ mean tweaking the API, it sounds like. But that's fine.

As a side note, isotopicDepletion has 74.4% code coverage.  Could be better, for sure, but not terrible: [file 1](https://coveralls.io/builds/64227929/source?filename=armi%2Fphysics%2Fneutronics%2FisotopicDepletion%2FcrossSectionTable.py), [file2](https://coveralls.io/builds/64227929/source?filename=armi%2Fphysics%2Fneutronics%2FisotopicDepletion%2FisotopicDepletionInterface.py).
